docs/src/components/Sidenav/index.tsx

```diff
@@ -13,7 +13,7 @@ interface WrapperProps {
 const StyledAsideWrapper = styled.aside<WrapperProps>`
   ${({ width, theme, shown }) => css`
     background: ${theme.orbit.paletteWhite};
-    padding: 10px 20px;
+    padding: 0 20px;
```

Comment:
it's a bit uncelar to me, when we should use `rem` and `px`, i understand cases when the css props are different, but now we have paddings in `px` and `rem` :) I think it's my part of the code, which was written before we started to use `rem` values in a project, but if you fixed it here, i think it will be better to replace as `rem` here too

Reply:
For stuff like padding it doesn't matter. In my CSS I prefer using rems when I don't know the exact value, or when it doesn't matter that much, and saying `1rem` is nicer to me than `16px` because using pixels makes me think that it has to be exactly this, but that's just my own personal preference.

I saw no reason to change your padding to rems, and we probably aren't ever going to change the root font size, so it both can be treated the same. But if you have strong opinions whether to use one or the other I'll follow your choice 😃

Reply:
okay, that's maybe just my approach, usually when i start to use `rem` for paddings/margins, i have all values or almost all set as `rems`, because it's convenient